app_dart/bin/server.dart

```diff
@@ -47,7 +47,12 @@ Future<void> main() async {
       '/api/debug/reset-pending-tasks': DebugResetPendingTasks(config, authProvider),
 
       '/api/public/build-status': GetBuildStatus(config),
-      '/api/public/get-benchmarks': GetBenchmarks(config),
+      '/api/public/get-benchmarks': CacheRequestHandler<Body>(
+        cache: await cacheService.redisCache(),
```

Comment:
You're calling `await cacheService.redisCache()` twice -- can you store it as a local variable and reuse it?

Reply:
Added